cli/azd/cmd/init.go

```diff
@@ -80,28 +82,34 @@ func (i *initFlags) Bind(local *pflag.FlagSet, global *internal.GlobalCommandOpt
 }
 
 type initAction struct {
-	azdCtx  *azdcontext.AzdContext
-	console input.Console
-	cmdRun  exec.CommandRunner
-	azCli   azcli.AzCli
-	gitCli  git.GitCli
-	flags   initFlags
+	azdCtx         *azdcontext.AzdContext
+	config         config.Config
```

Comment:
I think we likely want to introduce a service/manager for config... There's also a question of whether we should delay the loading until needed, or it should be loaded as part of action init. I prefer if we could lazy-load the config since it is not crucial to the action startup.

Reply:
`config.Config` is already a dependency on `account.Manager` so it needs to be constructed at action creation so you wouldn't be saving any time lazy loading it on the command.

The `Config` interface is very lightweight for the time being and feels overkill to introduce a Service / Manager.  `account.Manager` is already a wrapper around config for dealing with getting/setting account specific defaults like subscription & location.